src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPageVBWinForms.vb

```diff
@@ -223,8 +223,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                     'Set a flag so we don't keep trying to query for this property
                     _isMyApplicationPropertiesCached = True
 
-                    ' TODO: Remove this condition once VB property pages are fully supported on CPS
-                    If Not ProjectHierarchy.IsCapabilityMatch("PreventAutomaticMyApplication") Then
+                    'Only enable MyApplication when capability is present.
```

Comment:
Are you going to be changing legacy? The reason I made this an opt-out was because we want these enabled for legacy, and it doesn't have the `EnableMyApplication` capability. You should either add it to legacy, or put it back to opt-out and adjust the conditions as appropriate.